vendor/src/github.com/docker/distribution/digest/verifiers.go

```diff
@@ -1,8 +1,6 @@
 package digest
```

Comment:
I like the idea of a general purpose verifier for hash checking but this should have a constant time string comparison implementation for the checking in `Verified()`. Recommend using https://golang.org/pkg/crypto/subtle/#ConstantTimeCompare.